blueflood-core/src/main/java/com/rackspacecloud/blueflood/inputs/formats/JSONMetricsContainer.java

```diff
@@ -22,21 +22,21 @@
 import com.rackspacecloud.blueflood.types.Locator;
 import com.rackspacecloud.blueflood.types.Metric;
 import com.rackspacecloud.blueflood.utils.TimeValue;
-import org.apache.commons.lang.StringUtils;
 import org.codehaus.jackson.annotate.JsonIgnore;
 import org.joda.time.DateTime;
 
 import java.util.ArrayList;
 import java.util.List;
 import java.util.concurrent.TimeUnit;
 
+import static com.rackspacecloud.blueflood.inputs.formats.MetricValidator.*;
+
 public class JSONMetricsContainer {
+
     private final String tenantId;
     private final List<JSONMetric> jsonMetrics;
     private List<Metric> delayedMetrics;
     private static final long delayedMetricsMillis = Configuration.getInstance().getLongProperty(CoreConfig.DELAYED_METRICS_MILLIS);
-    private static final long pastDiff = Configuration.getInstance().getLongProperty( CoreConfig.BEFORE_CURRENT_COLLECTIONTIME_MS );
-    private static final long futureDiff = Configuration.getInstance().getLongProperty( CoreConfig.AFTER_CURRENT_COLLECTIONTIME_MS );
 
     public JSONMetricsContainer(String tenantId, List<JSONMetric> metrics) {
```

Comment:
@shintasmith @ChandraAddala 
The body of json for metrics and aggregated metrics are completely different, so we can't just have JSONMetric represent both normal and aggregated message (so the properties would be different for each class, with some common one such as tenantid).  The equivalent of JSONMetric is the AggregatedPayload.  

However, maybe I can move the createBundleList() call in HttpAggregatedMultiIngestionHandler to  JSONMetricsContainer and have both a 

List<JSONMetric> jsonMetrics,  and a 
List<AggregatedPayload> jsonMetricsAggregated Variable there, that can share the same validation code, and not have two classes refactor from a common base class. 

What do you guys think of that?